pcsx2/CDVD/cue_parser.h

```diff
@@ -0,0 +1,92 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2022  PCSX2 Dev Team
+ *
+ *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU Lesser General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with PCSX2.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ *  Original code from DusckStatoin by Stenznek
+ *  Modified by Weirdbeardgame for PCSX2 emu
+ */
+
+#pragma once
+#include "IopCommon.h"
+#include "CDVDaccess.h"
+
+#include <optional>
+#include <string_view>
+#include <utility>
+#include <vector>
+
+namespace Common {
+class Error;
+}
+
+namespace CueParser {
+
+// Defines in CDVDAccess for audio track types
+// using TrackMode = CDImage::TrackMode;
+
+// "MSF" would be SetSector in the cdr struct
```

Comment:
If it doesn't apply to our codebase, then remove everything about it. MSF has no meaning if it only exists on DuckStation.